src/vanilla/Templates/MethodJsonPipelineTemplate.cshtml

```diff
@@ -144,14 +144,48 @@ else
 <text>
       requestBodyMapper: requestModelMapper,
       requestBodyName: "@(Model.RequestBody.Name)",
+@if (Model.RequestBody.OperationType != null)
+{
+<text>
+      requestBodyType: msRest.OperationParameterType.@(Model.RequestBody.OperationType),
+</text>
+}
+      contentType: "@(Model.RequestContentType)",
```

Comment:
redundant?

Reply:
Nope, this is necessary for the regular request body scenario. The one on line 181-ish is required for the formdata scenario.